src/lib/swapper/swappers/CowSwapper/types.ts

```diff
@@ -45,3 +45,12 @@ export type CowSwapGetTradesResponse = {
 export type CowSwapGetTransactionsResponse = {
   status: 'presignaturePending' | 'open' | 'fulfilled' | 'cancelled' | 'expired'
 }[]
+
+export type AffiliateAppDataFragment =
+  | {
+      partnerFee: {
+        bps: string
+        recipient: string
+      }
+    }
+  | { partnerFee?: never }
```

Comment:
not sure what this union is about, but doesn't seem necessary or like it is doing anything of value
```suggestion
export type AffiliateAppDataFragment ={
  partnerFee?: {
    bps: string
    recipient: string
  }
}
```